simplq/src/components/pages/Admin/index.jsx

```diff
@@ -86,7 +86,7 @@ export default (props) => {
       <div className={styles['main-button-group']}>
         <div className={styles['admin-button']}>
           <StandardButton onClick={generateQrCOde} icon={<CropFreeIcon />} outlined>
-            Generate QrCode
+            Generate QR Code
```

Comment:
Do we want "Generate QR Code" or "Show QR Code"?

Reply:
I personally prefer generate.

Reply:
Cool. 👍